src/schema/validator.js

```diff
@@ -542,6 +542,67 @@ export class SchemaValidator {
       ),
       errors: true,
     });
+
+    const getPrivilegedPermissionsSet =
+      this.getPrivilegedPermissionsSet.bind(this);
+
+    validator.addKeyword({
+      keyword: 'postprocess',
+      validate: function validate(keywordSchemaValue, propValue) {
+        if (keywordSchemaValue === 'checkPrivilegePermission') {
+          const privilegedPermissions = getPrivilegedPermissionsSet(validator);
+          const found = new Set();
+
+          for (const permission of propValue) {
+            if (privilegedPermissions.has(permission)) {
+              found.add(permission);
+            }
+          }
+
+          if (found.size > 0) {
+            validate.errors = [
+              {
+                keyword: 'checkPrivilegedPermission',
```

Comment:
Probably a good idea to have a constant for this string value.

Reply:
yeah, I was waiting for the schema in the Phabricator revision to be finalized, before moving these into consts, I'll leave a TODO comment in this PR description comment for now.

Reply:
Well, in the end I decided to anticipate this, and just update it if it changes on the phabricator patch side, this is now part of this PR, changes introduced in [c3ba1b0](https://github.com/mozilla/addons-linter/pull/4307/commits/c3ba1b0792303b4b9500059277c79ffba5838515).